src/system.jl

```diff
@@ -99,6 +99,84 @@ struct Branch
     break_points::Tuple{Float64, Float64}
     "Price penalties for each of the break points of the branch (\$)"
     penalties::Tuple{Float64, Float64}
+    "Resistance of the transformer (pu)"
+    resistance::Float64
+    "Reactance of the transformer or branch (pu)"
+    reactance::Float64
+    "Boolean indicating whether the branch is a transformer"
+    is_transformer::Bool
+    "Ratio between the nominal winding one and two voltages of the transformer"
+    tap::Union{Missing, Float64}
+    "Phase shift angle (radians)"
+    angle::Union{Missing, Float64}
+end
+
+"""
+Constructor for a non-transformer branch which sets `is_transformer` to `false` and
+transformer specific variables to `missing`.
+"""
+function Branch(
```

Comment:
is this helpful as a user-facing docstring? 

right now i think this reads more as an internal comments about why we have this.

i'm wondering if it would be better, if we think users need help here, to _show_ this behaviour e.g.
something that shows the two (with some commentary)
```julia
line =        Branch("1", "A", "B", 10.0, 10.0, true, (100.0, 102.0), (5.0, 6.0), 1.0, 1.0)
transformer = Branch("4", "A", "C", 10.0, 10.0, true, (100.0, 102.0), (5.0, 6.0), 1.0, 1.0, 0.5, 30.0)
```